amgut/db/patches/0010.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE ag.external_survey ( 
+	external_survey_id   bigserial  NOT NULL,
+	external_survey      varchar  NOT NULL,
+	external_survey_description varchar  ,
```

Comment:
Might be good to add a timestamp field to this table, since the table is storing the contents of a particular survey that was retrieved at some particular point in time.